client/src/routes/Sessions/Sessions.tsx

```diff
@@ -95,7 +95,7 @@ const Sessions = () => {
   );
 
   return (
-    <Screen noTopBar backgroundColor={COLORS.PURE_WHITE}>
+    <Screen backgroundColor={COLORS.PURE_WHITE}>
```

Comment:
Shouldn't this be the default color in `Screen` or the default of the native app? Else we need to specify this everywhere? Right?

Reply:
Only session should have `PURE_WHITE`, all other routes should have `WHITE`, which is defined as default in `Screen`

Reply:
Ah ofc! Nice :)